components/UptimeMonitors/UptimeMonitors.js

```diff
@@ -2,6 +2,11 @@ import PropTypes from "prop-types";
 import UptimeMonitor from "../UptimeMonitor";
 
 const UptimeMonitors = ({ statusPage }) => {
+  // Display uptime monitors in alphabetical order
+  statusPage.uptime_monitors.sort((a, b) => {
+    return a.title.toLowerCase() < b.title.toLowerCase() ? -1 : 1;
```

Comment:
Can we use `localeCompare` instead? ( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare ) that takes into account non-ascii characters as well.

Reply:
I have switched to `localeCompare`.